metaci/build/models.py

```diff
@@ -660,6 +660,14 @@ def run_flow(self, project_config, org_config):
 
         flow_config = project_config.get_flow(self.flow)
 
+        if self.build.plan.role == "release":
+            flow_config["options"].update(
+                {
+                    "sandbox_date": self.build.release.sandbox_push_date,
+                    "production_date": self.build.release.production_push_date,
+                }
+            )
```

Comment:
@jofsky You're adding to the options for the flow as a whole, but we need to add to the options for one particular step in the flow (github_release_notes). Also, let's be careful not to mutate flow_config, because that might write back to the project config it came from.

I think what we want is:

    if self.build.plan.role == "release" and self.build.release:  # notice I changed the condition
        options = {
            "github_release_notes": {
                "sandbox_date": self.build.release.sandbox_push_date,
                "production_date": self.build.release.production_push_date
            }
        }
    else:
        options = {}

and then add options=options to the args when constructing FlowCoordinator below.